src/trans/cpu/internal/prfi1bad_mod.F90

```diff
@@ -88,8 +88,8 @@ SUBROUTINE PRFI1BAD(KM,PIA,PSPEC,KFIELDS,KFLDPTR)
     IFLD = KFLDPTR(JFLD)
     DO J=1,ILCM
       INM = IOFF+(ILCM-J)*2
-      PSPEC(IFLD,INM  ) = PSPEC(IFLD,INM  ) + PIA(J+2,IR)
```

Comment:
I would want to be very sure that we're not relying on the `+=` anywhere before changing the code like this. If we are relying on values already in `PSPEC` in any call to `PRFI1BAD`, the right fix for this issue might be to zero the 5 arrays (spvor, spdiv, spscalar, spsc2, spsc3b) at the beginning of `LTINV_CTLAD` ?